app/models/govbox_set.rb

```diff
@@ -84,7 +88,7 @@ def format_new_news
 
     @news_items&.results&.first(3)&.map do |news_item|
       {
-        title: news_item.title,
+        title: translate_highlights(news_item.title),
```

Comment:
I'm wondering if you could just do
```ruby
news_item.title.html_safe
```

Reply:
@ddzz can you provide your input on this?

Reply:
I don't think `#html_safe` will handle the special characters for highlighting.

Reply:
It does't seem to be a different functionality. Thanks @ddzz